lib/common/templates/scss/map-deep.template.js

```diff
@@ -12,14 +12,15 @@
  * @param {number} depth
  */
 function processJsonNode(obj, options, depth = 0) {
+  const { formatting } = options;
   let output = '';
   if (obj === null) {
     output += `''`;
   } else if (typeof obj === 'string') {
     output += `'${obj}'`;
   } else if (Object.hasOwn(obj, `${options.usesDtcg ? '$' : ''}value`)) {
     // if we have found a leaf (a property with a value) append the value
-    output += `$${obj.name}`;
+    output += `${formatting?.prefix ?? '$'}${obj.name}`;
```

Comment:
It looks like we are missing other `formatting` options as well here like `suffix`, should we add those in too?